### PR TITLE
add first draft of 2.17 what's new page

### PR DESCRIPTION
**Pull request by rosieyohannan**
Landing page for Server v2.17 with changelog items and upgrade instructions

### CODE REVIEW

**Comment by michelle-luna** [comment]
This looks good! Because it is a new document, we need to add a link in the sidebar (also for the new PDFs) in the https://github.com/circleci/circleci-docs/blob/master/jekyll/_data/sidenav.yml file

File: jekyll/_cci2/v.2.17-overview.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+categories: [getting-started]
+order: 1
+---
+
+This document provides a summary of features and product notes for the release of CircleCI Server v2.17.
+
+## What's New in Release 2.16
+
+* Workflows now has a Slack Integration! Users can choose to receive Slack notifications when their workflows complete.
+* Administrators can now restrict whichat organizations are allowed into their CircleCI installation. For more details on how to enable this feature, please see the User Management Section of the 2.17 Ops Manual.
```
Comment (michelle-luna) [inline jekyll/_cci2/v.2.17-overview.md]:
```suggestion
* Administrators can now restrict which organizations are allowed into their CircleCI installation. For more details on how to enable this feature, please see the User Management Section of the 2.17 Ops Manual.
```

File: jekyll/_cci2/v.2.17-overview.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+* We changed the renamed org flow so orgs that have been renamed will no longer result in errors going forward. Existing users who had applied workarounds for this use case will now no longer require said workarounds.
+* Workflows now take up less DB space and will be easier to maintain going forward.
+* Improved the cache in front of GraphQL API resulting in overall improved performance.
+* Added backpressure to avoid overwhelming nomad with requests, this will result in increased performance from existing nomad clusters.
+
+
+- PDF installation and operations documentation:
+   - *CircleCI v2.17 Installation Guide*
```
Comment (michelle-luna) [inline jekyll/_cci2/v.2.17-overview.md]:
```suggestion
   - [*CircleCI v2.17 Installation Guide*](https://circleci.com/docs/2.0/circleci-2-17-install-doc.pdf)
```

File: jekyll/_cci2/v.2.17-overview.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+   - *CircleCI v2.17 Operations Guide*
```
Comment (michelle-luna) [inline jekyll/_cci2/v.2.17-overview.md]:
```suggestion
   - [*CircleCI v2.17 Operations Guide*](https://circleci.com/docs/2.0/circleci-2-17-ops-guide.pdf)
```

File: jekyll/_cci2/v.2.17-overview.md (hunk 1 above)
Comment (rosieyohannan) [reply jekyll/_cci2/v.2.17-overview.md]: I made up a new word!

**Pull request merged**